File: server/models/Team.js
let mongoose = require('mongoose')
let Schema = mongoose.Schema

let teamSchema = new Schema({
  name: {
    type: String,
    required: true
  },
  color: {
    type: String,
    required: true
  },
  score: {
	type: Number,
	required: true
  }
  users: [ {type: Schema.ObjectId, ref: 'User'} ],
  tiles: [ {type: Schema.ObjectId, ref: 'Tile'} ]
})

let Team = mongoose.model('Team', teamSchema)

module.exports = Team

File: Client/GameComponents/Team.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GameComponents
{
    public class Team
    {
        public string name { get; set; }

        public string color { get; set; }

        public int score { get; set; }

        public User[] users { get; set; }

        public Tile[] tiles { get; set; }
    }
}
